### PR TITLE
Unit tests gen j

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/generate_unit_tests/test_case_builder.py [hunk 1]
```diff
@@ -0,0 +1,338 @@
+import ast as ast_mod
+import json
+import logging
+from pathlib import Path
+from typing import Dict, List
+
+from ordered_set import OrderedSet
```
Comment (dorschw) [inline demisto_sdk/commands/generate_unit_tests/test_case_builder.py:7]: trying to minimize dependencies, please consider using `dict.from_keys()`, then treating `dict.keys()` as an ordered set, see [here](https://www.programiz.com/python-programming/methods/dictionary/fromkeys)

File: Pipfile [hunk 2]
```diff
@@ -75,7 +74,8 @@ types-six="*"
 types-PyMySQL="*"
 types-protobuf="*"
 typing-extensions="*"
-
+klara = "==0.6.3"
```
Comment (dorschw) [inline Pipfile:77]: why do we use a locked version here?

Comment (JudahSchwartz) [reply Pipfile:77]: this pr was just for testing, I think you should add the comment to eyal's PR

File: demisto_sdk/commands/generate_unit_tests/test_case_builder.py (hunk 1 above)
Comment (JudahSchwartz) [reply demisto_sdk/commands/generate_unit_tests/test_case_builder.py:7]: same comment here, sorry

**Pull request closed without merging**